src/ITensorNetworks/peps.jl

```diff
@@ -183,22 +215,34 @@ function generate_inner_network(
     inner = inner_network(peps, peps_prime, peps_prime_ham, H_term.mpo, coords)
     network_list = vcat(network_list, [inner])
   end
-  inner = inner_network(peps, peps_prime)
-  network_list = vcat(network_list, [inner])
   return network_list
 end
 
-function generate_inner_network(
+function inner_networks(
   peps::PEPS,
   peps_prime::PEPS,
   peps_prime_ham::PEPS,
-  projectors::Array{<:ITensor,1},
+  projectors::Vector{<:ITensor},
```

Comment:
Actually just needs to be `Vector{ITensor}`. You would only need `Vector{<:ITensor}` if the ITensor type had a type parameter, but it does not.